### PR TITLE
Add grunt-contrib-csslint.

### LINKED ISSUE #52: Lint the stylesheets
We should have a linter for our CSS files.

Currently one of them has a problem:

> Error in parsing value for 'width'.  Declaration dropped. badge.css:41

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes #52.

The main problem is that csslint doesn't make the process stop when it errors. Also, the rules I added were just to make everything pass; more refactoring might be needed later.

### CODE REVIEW

**Comment by XhmikosR** [comment]
So @fmarier, see my comments above.

**Comment by fmarier** [comment]
> The main problem is that csslint doesn't make the process stop when it errors.

Does that mean that the test will pass even if the CSS isn't clean?

**Comment by XhmikosR** [comment]
Unfortunately, yes. I'm still looking into this, there was a time this didn't happen.

/CC @vladikoff: any thoughts?

**Comment by XhmikosR** [comment]
@fmarier: please check the branch. I remove the font as per your request, but I didn't leave the default font because on Windows Times New Roman is used and it just looks bad IMO.

Before:
1_old

After:
2_new

**Comment by vladikoff** [comment]
Why remove the Open Sans font?

**Comment by XhmikosR** [comment]
I don't know, for performance reasons. But it was @fmarier's suggestion not my choice :P

@vladikoff: I CC'ed you because grunt-contrib-csslint doesn't mark the process as failed when it fails. Do you have any idea about that?

File: public/style.css [hunk 1]
```diff
@@ -2,24 +2,13 @@
...
-  font-style: normal;
-  font-weight: 300;
-  src: local('Open Sans Light'), local('OpenSans-Light'), url('OpenSans-Light.woff') format('woff');
-}
-
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font: 400 16px/1.42857143 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (fmarier) [inline public/style.css]: Could we maybe keep `Open Sans` in the list (without providing it as a downloadable `woff` file) for those people lucky enough to have it installed locally already?

Comment (XhmikosR) [inline public/style.css]: I think we could try woff2 maybe too?

**Comment by XhmikosR** [comment]
So @fmarier, what should I do with this? Keep the font so that we get csslint merged?

**Comment by fmarier** [comment]
> what should I do with this? Keep the font so that we get csslint merged?

Yeah, maybe we should leave the fonts alone and fix only what's needed to get csslint merged.

**Comment by mozfreddyb** [comment]
Hey, my previous merge created a (pretty small) conflict. Please resolve and keep the font and I'll be happy to get this landed.

**Comment by XhmikosR** [comment]
@mozfreddyb: rebased.

**Comment by XhmikosR** [comment]
@mozfreddyb @fmarier: so, can this be merged or is something else you'd like?

**Comment by fmarier** [comment]
Thanks @XhmikosR!

**Pull request merged**